rtmp-client/src/main/java/net/butterflytv/rtmp_client/RtmpClient.java

```diff
@@ -13,6 +13,17 @@
 
 
     private final static int OPEN_SUCCESS = 1;
+
+    /**
+     * RTMP read has received an EOF or READ_COMPLETE from the server
+     */
+    public final static int RTMP_READ_DONE = -1;
+
+    /**
+     * No error
+     */
+    public final static int RTMP_SUCCESS = 0;
```

Comment:
* Would this constant ever be needed externally? I don't think so, in which case it can be private
* Is `OPEN_SUCCESS` (above) ever used? Remove if not.

Reply:
Done